x-pack/platform/plugins/shared/onechat/public/application/components/conversations/new_conversation_button.tsx

```diff
@@ -27,12 +29,17 @@ export const NewConversationButton: React.FC<{}> = () => {
     if (isNewConversation) {
       cleanConversation();
     }
+    if (isEmbeddedContext) {
+      setConversationId?.(undefined);
+    }
   };
 
   const buttonProps = isDisabled
     ? {
         disabled: true,
       }
+    : isEmbeddedContext
+    ? {}
```

Comment:
We should refactor this at some point, this was a quick fix (can refactor when we have actual designs).